scripts/dashboard-version

```diff
@@ -0,0 +1,3 @@
+# This file is updated by running scripts/update-dashboard.sh
+# Don't edit it manullay
+2023.12.18.1
```

Comment:
tidb-dashboard version will change to `x.y.z` (likes `7.6.0`) style in the future, same as TiDB/TiKV/PD.